frontend/src/components/Project/ViewProject.tsx

```diff
@@ -0,0 +1,205 @@
+import * as React from 'react';
+import { DataGrid, GridColDef } from '@material-ui/data-grid';
+import { useEffect, useState } from 'react';
+import { getAllBusinessProject } from '../../services/ProjectAPI';
+import { Button, Grid, Link } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+
+const ViewProject: React.FC = () => {
+  const [projectList, setProjectList] = useState<any>([]);
+  const history = useHistory();
+
+  interface Sale {
+    amount: number;
+  }
+
+  interface Data {
+    id: number;
+    title: string;
+    email: string;
+    description: string;
+    status: string;
+    serviceType: string;
+    createdDate: string;
+    followUpDate: string;
+    deadlineDate: string;
+    leadSource: string;
+    leadRanking: string;
+    leadCredit: string;
+    sale: Sale;
+  }
+  interface DataDisplay {
+    id: number;
+    title: string | any;
+    email: string;
+    description: string;
+    status: string;
+    service: string;
+    createdDate: string;
+    followUpDate: string;
+    deadlineDate: string;
+    leadSource: string;
+    leadRanking: string;
+    leadCredit: string;
+    sale: number;
+  }
+
+  const [select, setSelection] = useState<any>('');
+  const handleRowSelection = (id: any) => {
+    setSelection(id);
+  };
+
+  const clickAddProject = () => {
+    history.push('/project');
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const projects = await getAllBusinessProject(1);
+      const display: DataDisplay[] = [];
+      projects.data.forEach((element: Data) => {
+        const amount: number = element.sale.amount;
+        const createdDate = element.createdDate.split('T');
+        const followUpDate = element.followUpDate.split('T');
+        const deadlineDate = element.deadlineDate.split('T');
+        display.push({
+          id: element.id,
+          title: { id: element.id, title: element.title },
+          email: element.email,
+          description: element.description,
+          status: element.status.charAt(0) + element.status.substring(1).toLowerCase(),
+          service: element.serviceType,
+          createdDate: createdDate[0],
+          followUpDate: followUpDate[0],
+          deadlineDate: deadlineDate[0],
+          leadSource: element.leadSource,
+          leadCredit: element.leadCredit,
+          leadRanking: element.leadRanking,
+          sale: amount,
```

Comment:
you can just call `element.sale.amount` to keep it consistent with the rest and you can remove line 61.

Reply:
sure